src/components/SignInButton.js

```diff
@@ -1,24 +1,34 @@
+import { useEffect } from 'react'
 import { h } from 'react-hyperscript-helpers'
 import { Clickable } from 'src/components/common'
+import { spinner } from 'src/components/icons'
+import { authSettled } from 'src/libs/auth'
+import { authStore } from 'src/libs/state'
 import * as Utils from 'src/libs/utils'
 
 
-const SignInButton = (props = {}) => {
-  Utils.useOnMount(() => {
-    window.gapi.signin2.render('signInButton', {
-      scope: 'openid profile email',
-      width: 250,
-      height: 56,
-      longtitle: true,
-      theme: 'light',
-      prompt: 'select_account'
-    })
-  })
+const SignInButton = (props = { theme: 'light' }) => {
+  const auth = Utils.useStore(authStore)
+
+  const isGoogleAuthInitialized = authSettled(auth)
+
+  useEffect(() => {
+    if (isGoogleAuthInitialized) {
+      window.gapi.signin2.render('signInButton', {
+        scope: 'openid profile email',
+        width: 250,
+        height: 56,
+        longtitle: true,
+        theme: props.theme,
```

Comment:
With the above, I think you can:
```suggestion
        theme,
```

Reply:
I just found that I somehow didn't push all of my changes. :( Update to SignInButton incoming...